test/http.test.js

```diff
@@ -89,4 +90,66 @@ describe('http', function () {
       })
     })
   })
+
+  describe('http (running on https protocol)', function () {
+    it('Creates a seneca server running on port 8000 https and expects hex to be equal to #FF0000', function (done) {
+      process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+      function color () {
+        this.add('color:red', function (args, done) {
+          done(null, {hex: '#FF0000'})
+        })
+      }
+
+      Seneca({
+        log: 'silent'
+      })
+        .use('../transport')
+        .use(color)
+        .listen({
+          type: 'web',
+          port: 8000,
+          host: '127.0.0.1',
+          protocol: 'https',
+          serverOptions: {
+            keyPemPath: './ssl/key.pem',
```

Comment:
once the server options are changed, you can use `key` and `cert` and copy the values from the wreck test if you like